src/uwtools/config/formats/base.py

```diff
@@ -14,7 +14,7 @@
 from uwtools.config import jinja2
 from uwtools.config.support import INCLUDE_TAG, depth, log_and_error
 from uwtools.exceptions import UWConfigError
-from uwtools.logging import log
+from uwtools.logging import INDENT, log
```

Comment:
A standard indentation value for log messages is now defined in the `uwtools.logging` module and used where appropriate.